apps/demos/Demos/Chat/Customization/jQuery/data.js

```diff
@@ -0,0 +1,53 @@
+const getTimestamp = function(date, offsetMinutes = 0) {
+    return date.getTime() + offsetMinutes * 60000;
+}
+
+const date = new Date();
+date.setHours(0, 0, 0, 0);
+
+const currentUser = {
+    id: "c94c0e76-fb49-4b9b-8f07-9f93ed93b4f3",
+    name: "John Doe",
+};
+
+const supportAgent = {
+    id: "d16d1a4c-5c67-4e20-b70e-2991c22747c3",
+    name: "Support Agent",
+    avatarUrl: "../../../../images/petersmith.png",
+};
+
+const messages = [
+    {
+        timestamp: getTimestamp(date, -9),
+        author: supportAgent,
+        text: "Hello, John!\nHow can I assist you today?"
+    },
+    {
+        timestamp: getTimestamp(date, -7),
```

Comment:
Indentions of the file incorrect

Reply:
https://github.com/DevExpress/DevExtreme/pull/28441/commits/7abeedba5198baaf75acd81fdb9e03e779b24d94